corehq/apps/reports/templates/reports/filters/datespan.html

```diff
@@ -8,6 +8,7 @@
                    id="filter_range"
                    class="date-range-picker form-control report-filter-datespan"
                    value="{{ datespan.startdate|date:'Y-m-d' }} to {{ datespan.enddate|date:'Y-m-d' }}"
+                   placeholder="{% trans "Show All Dates"|escapejs %}"
```

Comment:
curious what this changed / why it was necessary?

Reply:
Good point. It doesn't seem to do anything, so I'll remove this line. Thanks for catching that!